test/mixins.py

```diff
@@ -245,6 +245,14 @@ def bigflow_spawn(self, cmd, **kwargs):
         return self.subprocess_spawn(cmd, **kwargs)
 
 
+class IteratorAssertionsMixin(Mixin):
+    def assertEmpty(self, iterator: Iterator):
+        self.assertFalse(list(iterator), msg="Iterator should be empty")
```

Comment:
IMO those functions are not super needed (they save us from typing a few chars), and may be removed.

Reply:
`assertEmpty()` is way more readable then `assertFalse(list(...))`, so I'll insist on leaving them.

Reply:
This is `nit` for me, so you really could leave those functions if you think they are useful.

However I'm not fully agree that it is more readable, at least not in a Python.
"Readable" means "most of readers will understand this quickly and without extra brain-work".

There is an existing question for similar problem on SO https://stackoverflow.com/questions/33216488/how-to-assert-that-an-iterable-is-not-empty-on-unittest (assert list is empty, not an iterator).

**115** users vote for just using `self.assertTrue(d)` (and rely on well known python [truth/falsy](https://docs.python.org/3/library/stdtypes.html#truth-value-testing) concept).
**8** users vote for using/adding `self.assertEmpty`.